### PR TITLE
feat: add account user and pending invites table

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

File: assets/svelte/settings/AccountSettings.svelte [hunk 1]
```diff
@@ -93,59 +113,163 @@
...
-        >
-          Delete Account
+    <div class="bg-card text-card-foreground rounded-lg border shadow-sm">
+      <div class="flex p-6 justify-between">
+        <h3 class="text-lg font-semibold leading-none tracking-tight">
+          Invitations
+        </h3>
+        <Button>
+          <Plus class="h-4 w-4 mr-2" />
+          Invite user
```
Comment (acco) [inline assets/svelte/settings/AccountSettings.svelte:215]: Hide when there are 0 pending users

Comment (yordis) [reply assets/svelte/settings/AccountSettings.svelte:215]: How could you invite new users, then?

Comment (yordis) [reply assets/svelte/settings/AccountSettings.svelte]: done

**Comment by yordis** [comment]
@RTLS share some love over here btw, I will continue the work in the modal after this one

**Pull request merged**